### ISSUE TITLE
I'm a ghost. It's from log(n), not log(log(n))

### ISSUE THREAD

**Issue by gabboman** [in-memoriam/in-memoriam#1]
The time to access when you have more than 100 quantillions gets to log(log(n)) instead only K.

**Comment by ewnavilae** [comment]
I just confirmed this bug and I am a zombie so I guess it's not his machine.

**Comment by sinisa229** [comment]
Yeah, this issue popped up on our performance tests, but it only happens with ghosts, zombies, vampires and viruses. 

No reports of this issue have been made by permanently and properly dead(PPD). 

Whats your permDead setting? Try increasing it to **permDead=-5&*#**.

I hope this helps.
